notebooks/en/rag_with_hf_and_milvus.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<p>"...configure your Hugging Face User Access Token, and set it in your environment variables because we will use a LLM from the Hugging Face Hub."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>